app/src/androidTest/java/com/readtracker/android/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,29 @@
+package com.readtracker.android;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by anthonymonori on 05/10/15.
```

Comment:
Please drop all headers. This information is irrelevant 99.99% of the cases (and available through git log when it isn't :))

Reply:
@anthonymonori Is this an auto-generated example test that Android Studio creates? If so, it's probably not worth keeping unless you think the test is actually valuable (and then it's probably worth renaming it).

Reply:
@christoffer I will remove the boilerplate header added by Android Studio - I will also update my preferences so that it won't add that to every new file I add. I have that on my work profile, but for this project I switched back to default and haven't noticed it. Thanks for catching it!

Reply:
@peter-tackage Yes, it is the auto-generated androidTest class by AS. I would like to keep it here though, because this will ensure that the emulator tests (functional tests) are able to run. If we would throw this out, we won't catch when our emulators in the CI setup are not functional anymore. I also hope to add more functional tests as we do more work on this, and then we'll throw this one out!

Reply:
Fixed this in fe5b3d71a10967c157d09e9e39b63e6f876f2d29